### PR TITLE
I1405 hash of hash logging

### LINKED ISSUE #1405: Add debug logging about hashOfHash
Should add logging to describe how `hashOfHash` is determined. Log each of the component hashes and their influences.

### PR DESCRIPTION
**Pull request by laneb**
Resolves #1405. Will be really helpful to debug some issues we've been seeing recently.

### CODE REVIEW

**Comment by AliTajeldin** [comment]
@laneb this failed CI.

**Comment by laneb** [comment]
@AliTajeldin Travis is being a PIA
```
213.65s$ make test-quick
sbt test
Downloading sbt launcher for 0.13.15:
  From  http://repo.typesafe.com/typesafe/ivy-releases/org.scala-sbt/sbt-launch/0.13.15/sbt-launch.jar
    To  /home/travis/.sbt/launchers/0.13.15/sbt-launch.jar
Download failed. Obtain the jar manually and place it at /home/travis/.sbt/launchers/0.13.15/sbt-launch.jar
make: *** [test-scala] Error 1
```
will rerun it and see what happens.

File: src/main/scala/org/tresamigos/smv/SmvDataSet.scala [hunk 1]
```diff
@@ -380,8 +394,16 @@ abstract class SmvDataSet {
...
   /** Has the result of this data set been persisted? */
-  private[smv] def isPersisted: Boolean =
-    Try(new FileIOHandler(app.sparkSession, moduleCsvPath()).readSchema()).isSuccess
+  private[smv] def isPersisted: Boolean = {
+    val csvPath = moduleCsvPath()
+    val ioHandler = new FileIOHandler(app.sparkSession, csvPath)
+    val res = Try(ioHandler.readSchema()).isSuccess
+
+    if (res) 
+      app.log.debug("Couldn't find ${ioHandler.fullSchemaPath} - ${fqn} not persisted")
```
Comment (ninjapapa) [inline src/main/scala/org/tresamigos/smv/SmvDataSet.scala]: Should it be `no res` here?

Comment (laneb) [reply src/main/scala/org/tresamigos/smv/SmvDataSet.scala]: fixed

**Pull request merged**